src/node-lib/curriculum-api-2023/queries/lessonOverview/lessonOverview.query.ts

```diff
@@ -265,11 +266,14 @@ const lessonOverviewQuery =
     const pathways = canonicalLesson ? getPathways(res) : [];
 
     lessonBrowseDataByKsSchema.parse(browseDataSnake);
-    lessonContentSchema.parse(contentSnake);
+    lessonContentSchema.parse({ ...contentSnake, phonics_outcome: null });
```

Comment:
why are we setting it to null?

Reply:
Phonic data is only on the new lesson content mv and the current lesson overview shares types with beta overview page

Reply:
gotcha thanks, do we have that documented anywhere atm?

Reply:
I'll drop a comment on it, there's gonna be more of this until some of these things become published i think